Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -440,4 +429,48 @@ def check_for_duplicate_pull_requests(formula)
       EOS
     end
   end
+
+  def alias_update_pair(formula, new_formula_version)
+    versioned_alias = formula.aliases.grep(/.@\d/).first
+    return if versioned_alias.nil?
+
+    name, old_alias_version = versioned_alias.split("@")
+    if old_alias_version.split(".").length == 1
+      new_alias_version, = *new_formula_version.to_s.match(/^\d+/)
+    else
+      new_alias_version, = *new_formula_version.to_s.match(/^\d+\.\d+/)
+    end
+    return if Version.create(new_alias_version) <= Version.create(old_alias_version)
+
+    [versioned_alias, "#{name}@#{new_alias_version}"]
+  end
+
+  def run_audit(formula, alias_rename)
+    if args.dry_run?
+      if args.no_audit?
+        ohai "Skipping `brew audit`"
+      elsif args.strict?
+        ohai "brew audit --strict #{formula.path.basename}"
+      else
+        ohai "brew audit #{formula.path.basename}"
+      end
```

Comment:
Could add a `return` after the `end` and avoid the `else` nesting below.

Reply:
Sure, given that it is its own function. Also this could possible be done in `inreplace_pairs()` too I guess.